docs/getting-started/cases-req.asciidoc

```diff
@@ -23,11 +23,14 @@ To grant access to cases, set the {kib} space privileges for the *Cases* and *Ac
 | Give full access to manage cases
 a|
 * `All` for the *Cases* feature
-* `All` for the *Action and Connector* feature (go to *Management > Actions and Connectors* to set this)
+* `All` for the *{connectors-feature}* feature (go to *Management > {connectors-feature}* to set this)
 
-NOTE: Roles without `All` *Actions and Connectors* feature privileges cannot create, add, delete, or modify case connectors.
+NOTE: Roles without `All` *{connectors-feature}* feature privileges cannot create, add, delete, or modify case connectors.
 
 | Give view-only access for cases | `Read` for the *Cases* feature
+
+| Give access to view and delete cases | `Read` for the *Cases* feature with the deletion sub-feature enabled
```

Comment:
```suggestion
| Give access to view and delete cases | `Read` for the *Cases* feature and the *Delete* sub-feature selected
```

Reply:
Mainly making this suggestion so users know exactly what to look for. Also changed "enabled" to "selected" to align the verb it with our [UI writing tips](https://wiki.elastic.co/pages/viewpage.action?pageId=105914797#Writingforandabouttheuserinterface(UI)-Choosingtherightword).